pagination/header.go

```diff
@@ -15,11 +15,11 @@ func header(u *url.URL, rel string, limit, offset int) string {
 	return fmt.Sprintf("<%s>; rel=\"%s\"", u.String(), rel)
 }
 
-// Header returns an http header map, with a single key, "Link". The "Link" header is used in pagination where backwards compatibility is required.
-// The Link header will contain links any combination of the first, last, next, or previous (prev) pages in a paginated list (given a limit and an offset, and optionally a total).
+// Header returns string to be used in an http header map for pagination where backwards compatibility is required.
+// The value returned will contain links any combination of the first, last, next, or previous (prev) pages in a paginated list (given a limit and an offset, and optionally a total).
 // If total is not set, then no "last" page will be calculated.
 // If no limit is provided, then it will default to 1.
-func Header(u *url.URL, total int, limit, offset int) http.Header {
+func Header(u *url.URL, total int, limit, offset int) string {
```

Comment:
In that case it would probably make more sense to have the signature initially proposed (`Header(http.ResponseWriter, *url.URL, total, limit, offset int)`). What do you think?

Reply:
Would that make it harder to test? I haven't looked into mocking an http.ResponseWriter.

Reply:
`httptest.NewRecorder` :)